storybook/qmlTests/tests/tst_CurrencyAmountInput.qml

```diff
@@ -0,0 +1,182 @@
+import QtQuick 2.15
+import QtTest 1.15
+
+import Storybook 1.0
+import shared.controls 1.0
+
+Item {
+    id: root
+    width: 600
+    height: 400
+
+    Component {
+        id: componentUnderTest
+        CurrencyAmountInput {
+            id: input
+            anchors.centerIn: parent
+        }
+    }
+
+    QtObject {
+        id: defaults
+
+        readonly property int decimals: 2
+        readonly property string currencySymbol: "USD"
+    }
+
+    TestCase {
+        name: "CurrencyAmountInput"
+        when: windowShown
+
+        property CurrencyAmountInput controlUnderTest: null
+
+        function init() {
+            controlUnderTest = createTemporaryObject(componentUnderTest, root)
+            controlUnderTest.locale = "en_US.utf8"
```

Comment:
```suggestion
            controlUnderTest.locale = "en_US"
```
the `.utf8` part is ignored from what I know so it may be confusing why it's here.

Reply:
Fixed